Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/Competition/BlueLeftAutonomousCompetition.java

```diff
@@ -0,0 +1,53 @@
+package org.firstinspires.ftc.team6220_PowerPlay.Competition;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.team6220_PowerPlay.AprilTagDetect;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+@Autonomous(name = "BlueLeft", group = "Competition")
+public class BlueLeftAutonomousCompetition extends AprilTagDetect {
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initialize();
+
+        servoGrabber.setPosition(0.45);
+        driveTurntable(1.0, Constants.TURNTABLE_DEFAULT_POSITION);
+        int signal = detectAprilTag();
+
+        IMUOriginalAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+        driveWithIMU(0.25, 0.0, 0.0);
+        sleep(1688);
+
+        IMUOriginalAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+        driveWithIMU(0.0, 0.0, 0.0);
+        sleep(500);
+
+        switch (signal) {
+            case 0:
+                IMUOriginalAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
+                driveWithIMU(0.0, -0.25, 0.0);
+                sleep(1550);
+
+                IMUOriginalAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
```

Comment:
In autonomous we need to do it every time, but in teleop it only needs to be called one. Our autonomous code didn't get a lot of time to be tested and is really bad, but it will get rewritten and then driveWithIMU won't need line 36 to be called every time